build_engine_test_images/run.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [[ -z "$TF_VAR_build_engine_aws_access_key" ]]; then
+    echo "Must provide TF_VAR_build_engine_aws_access_key in environment" 1>&2
+    exit 1
+fi
+
+if [[ -z "$TF_VAR_build_engine_aws_secret_key" ]]; then
+    echo "Must provide TF_VAR_build_engine_aws_secret_key in environment" 1>&2
+    exit 1
+fi
+
+if [[ -z "$TF_VAR_docker_id" ]]; then
+    echo "Must provide TF_VAR_docker_id in environment" 1>&2
+    exit 1
+fi
+
+if [[ -z "$TF_VAR_docker_password" ]]; then
+    echo "Must provide TF_VAR_docker_password in environment" 1>&2
+    exit 1
+fi
+
+if [[ -z "$TF_VAR_docker_repo" ]]; then
+    echo "Must provide TF_VAR_docker_repo in environment" 1>&2
+    exit 1
+fi
+
+# Build amd64 images
+export TF_VAR_build_engine_arch="amd64"
+export TF_VAR_build_engine_aws_instance_type="t2.micro"
+terraform -chdir=${TF_VAR_tf_workspace}/terraform/aws/ubuntu init
+terraform -chdir=${TF_VAR_tf_workspace}/terraform/aws/ubuntu apply -auto-approve -no-color
+./scripts/cleanup.sh
```

Comment:
Can we continue if there are errors here? Similar below

Reply:
Yes I add `|| true` after those command

Reply:
- I mean, if the failure of this part (including the 1st cleanup failure) would affect the 2nd part execution? Or is it meaningful to continue after the 1st part failure?
- I don't think we need `|| true` since by default the script is using `set +e`.

Reply:
HI @shuo-wu ,

As discussed, I added `set -e` and `trap ./scripts/cleanup.sh EXIT` make sure AWS resource will cleanup after script stopped